Common/Statistics/PortfolioStatistics.cs

```diff
@@ -300,5 +297,30 @@ private static decimal GetAnnualVariance(List<double> performance, int tradingDa
             var variance = performance.Variance();
             return variance.IsNaNOrZero() ? 0 : (decimal)variance * tradingDaysPerYear;
         }
+
+        /// <summary>
+        /// Tracking error volatility (TEV) statistic - a measure of how closely a portfolio follows the index to which it is benchmarked
+        /// </summary>
+        /// <remarks>If algo = benchmark, TEV = 0</remarks>
+        /// <param name="listPerformance">Double collection of algorithm performance values</param>
+        /// <param name="listBenchmark">Double collection of benchmark performance values</param>
+        /// <param name="tradingDaysPerYear">Trading days per year for the assets in portfolio</param>
+        /// <returns>Tracking error value</returns>
+        private static decimal GetTrackingError(List<double> listPerformance, List<double> listBenchmark, int tradingDaysPerYear = 252)
```

Comment:
I'd suggest not adding this new method an use the existing fixed implementation at `Statistics`, we can add the optional `tradingDaysPerYear` argument